src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/PrisonerIndexResource.kt

```diff
@@ -157,7 +99,5 @@ class PrisonerIndexResource(
     ]
   ) fun indexQueueHousekeeping() {
     prisonerIndexService.indexingComplete(ignoreThreshold = false)
-    queueAdminService.transferIndexMessages()
```

Comment:
not quite sure where these have gone too?

Reply:
These are replaced by the `/queue-admin/retry-all-dlqs` endpoint from the queue lib.